tsl/test/expected/continuous_aggs_multi.out

```diff
@@ -386,10 +386,126 @@ select * from cagg_1 where timed = 18 ;
+AS
+SELECT time_bucket(2, timeval), max(col2)
+FROM continuous_agg_test_ignore_invalidation_older_than
+GROUP BY 1;
+refresh materialized view cagg_iia1;
+INFO:  new materialization range for public.continuous_agg_test_ignore_invalidation_older_than (time column timeval) (14)
+INFO:  materializing continuous aggregate public.cagg_iia1: nothing to invalidate, new range up to 14
+select * from cagg_iia1 order by 1;
+ timed | cnt 
+-------+-----
+    10 |   2
+    12 |   1
+(2 rows)
+
+refresh materialized view cagg_iia2;
+INFO:  new materialization range for public.continuous_agg_test_ignore_invalidation_older_than (time column timeval) (14)
+INFO:  materializing continuous aggregate public.cagg_iia2: nothing to invalidate, new range up to 14
+select * from cagg_iia2 order by 1;
+ timed | maxval 
+-------+--------
+    10 |      5
+    12 |      7
+(2 rows)
+
+refresh materialized view cagg_iia3;
+INFO:  new materialization range for public.continuous_agg_test_ignore_invalidation_older_than (time column timeval) (14)
+INFO:  materializing continuous aggregate public.cagg_iia3: nothing to invalidate, new range up to 14
+select * from cagg_iia3 order by 1;
+ timed | maxval 
+-------+--------
+    10 |      5
+    12 |      7
+(2 rows)
+
+INSERT INTO continuous_agg_test_ignore_invalidation_older_than VALUES
+(1, -4, 1), (5, -3, 5), (10, -3, 9), (12,3,19);
+refresh materialized view cagg_iia1;
+INFO:  new materialization range not found for public.continuous_agg_test_ignore_invalidation_older_than (time column timeval): not enough new data past completion threshold (14)
+INFO:  materializing continuous aggregate public.cagg_iia1: processing invalidations, no new range
+--should see change to the 12, 10 bucket but not the 4 or 1 bucket
+select * from cagg_iia1 order by 1;
+ timed | cnt 
+-------+-----
+    10 |   3
+    12 |   2
+(2 rows)
+
+--should see change to the 12, 10 and 4 bucket but not the 1 bucket
+refresh materialized view cagg_iia2;
+INFO:  new materialization range not found for public.continuous_agg_test_ignore_invalidation_older_than (time column timeval): not enough new data past completion threshold (14)
+INFO:  materializing continuous aggregate public.cagg_iia2: processing invalidations, no new range
+select * from cagg_iia2 order by 1;
+ timed | maxval 
+-------+--------
+     4 |      5
+    10 |      9
+    12 |     19
+(3 rows)
+
+--sees no changes
+refresh materialized view cagg_iia3;
+INFO:  new materialization range not found for public.continuous_agg_test_ignore_invalidation_older_than (time column timeval): not enough new data past completion threshold (14)
+INFO:  materializing continuous aggregate public.cagg_iia3: nothing to invalidate, no new range
+INFO:  materializing continuous aggregate public.cagg_iia3: no new range to materialize or invalidations found, exiting early
+select * from cagg_iia3 order by 1;
+ timed | maxval 
+-------+--------
+    10 |      5
+    12 |      7
+(2 rows)
+
```

Comment:
would be good to add select from continuous_aggs_hypertable_invalidation_log here to show the modification_time before/after the new row is picked up.